hummingbot/connector/exchange/genius_yield/genius_yield_exchange.py

```diff
@@ -167,24 +174,20 @@ async def _place_order(self,
         type_str = GeniusYieldExchange.genius_yield_order_type(order_type)
         side_str = CONSTANTS.SIDE_BUY if trade_type is TradeType.BUY else CONSTANTS.SIDE_SELL
         symbol = await self.exchange_symbol_associated_to_pair(trading_pair=trading_pair)
```

Comment:
Needs to be double checked wheter trading_pair is appropriate here. Might be correct to use asset_ticker instead